.github/workflows/integration.yml

```diff
@@ -0,0 +1,73 @@
+name: Integration testing
+env:
+  PROJECT_ROOT: /root/go/src/github.com/infrawatch/sg-core
+
+  QDR_IMAGE: quay.io/interconnectedcloud/qdrouterd:1.15.0
+  QDR_VOLUME: "--volume=${{ github.workspace }}/ci/service_configs/qdr:/etc/qpid-dispatch:ro"
+  QDR_PORT: "-p 5666:5666"
+
+  LOKI_IMAGE: grafana/loki:2.1.0
+  LOKI_VOLUME: "--volume=${{ github.workspace }}/ci/service_configs/loki:/etc/loki:ro"
+  LOKI_PORT: "-p 3100:3100"
+
+  TEST_IMAGE: centos:8
```

Comment:
Can this be moved to ubi8:latest instead? Otherwise we should consider moving to centos stream

Reply:
I'd have preferred ubi8 here since that's what we're going to be building against downstream, but since I called out stream I guess this is ok.